lib/manager/components/DeploymentViewer.js

```diff
@@ -168,6 +216,9 @@ export default class DeploymentViewer extends Component<Props, State> {
       {east: 10, north: 10, west: -10, south: -10}
     const boundsTooLarge = east - west > BOUNDS_LIMIT || north - south > BOUNDS_LIMIT
     const bounds = this._getBounds()
+    const serverDeployedTo = deployment.deployedTo
```

Comment:
Would prefer having this be a boolean variable instead of possibly null.

Reply:
Can you clarify? Are you referring to `serverDeployedTo`? This variable (type `OtpServer`) is used below to get the name of the server.

Reply:
Oh, whoops, nevermind.